File: c++/pattern.js
function pattern() {
  const mainarr = [1, 2, 3, 4, 5];
  mainarr.forEach((value) => {
    let line = "";
    for (let i = 0; i < value; i++) {
      line += "x";
    }
    console.log(line);
  });
}
pattern();

File: c++/pattern.lua
function Pattern()
	A = { 1, 2, 3, 4, 5 }
	for _, x in pairs(A) do
		for _ = 1, x, 1 do
			io.write("x")
		end
		io.write("\n")
	end
end

print(Pattern())
